cfme/tests/infrastructure/test_provisioning.py

```diff
@@ -204,5 +231,29 @@ def verify():
                 in normalize_text(mail["subject"]),
                 smtp_test.get_emails())) == len(vm_names)
         )
-
     wait_for(verify, message="email receive check", delay=5)
+
+
+@pytest.mark.parametrize("approve", [True, False], ids=["approved", "denied"])
+def test_provision_approval_msg(
+        setup_provider, provider, vm_name, request, approve, provision_request):
+    template, prov_data = provision_request
+    prov_data['num_vms'] = "2"
+    if approve:
```

Comment:
REQUIRED: you do the same call in both cases when approve True and False. There is definitely something wrong.